docs/BasicCommunication/SystemRequest/index.md

```diff
@@ -32,7 +35,8 @@ SDL validates all SystemRequests sent from the mobile app and returns DISALLOWED
 
 |Name|Type|Mandatory|Additional|
 |:---|:---|:--------|:---------|
-|requestType|[Common.RequestType](../../common/enums/#requesttype)|true||
+|requestType|[Common.RequestType](../../common/enums/#requesttype)|true||  
+|requestSubType|String|false|maxlength="255"|This parameter is filled for supporting OEM proprietary data exchanges.|
```

Comment:
Missed this before, but it's the same case here, there is no 5th column

Reply:
fixed https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/98/commits/ab7e9a293fa03fc5cae1ad15ff8809825e8196b3